source/features/resolve-comments.js

```diff
@@ -0,0 +1,74 @@
+import select from 'select-dom';
+import {h} from 'dom-chef';
+import {getUsername} from '../libs/utils';
+import * as icons from '../libs/icons';
+
+const RESOLVED_TEXT_REGEX = /^Comment resolved by @([a-zA-Z0-9-]+)/;
+
+export default () => {
+  for (const comment of select.all('.comment.timeline-comment')) {
+    const body = select('.comment-body', comment);
+    const isResolved = RESOLVED_TEXT_REGEX.test(body.innerText);
+
+    if (isResolved) {
+      const commentContainer = select('.edit-comment-hide', comment);
+      commentContainer.style.display = 'none';
+      const toggle = (
+        <button
+          type="button"
+          class="btn-link"
+          onClick={() => {
+            if (commentContainer.style.display === 'none') {
+              commentContainer.style.display = '';
+              toggle.innerHTML = icons.fold().outerHTML;
+            } else {
+              commentContainer.style.display = 'none';
+              toggle.innerHTML = icons.unfold().outerHTML;
```

Comment:
No need to pass through HTML; use `toggle.firstChild.replaceWith(icons.unfold())` (or `firstElementChild`)